app/players/elves/elf.py

```diff
@@ -0,0 +1,13 @@
+from ..player import Player
+
+
+class Elf(Player):
```

Comment:
The `Elf` class does not implement all abstract methods from the `Player` class, which is why it should inherit `ABC` as well.